packages/api/app/routes.js

```diff
@@ -83,5 +86,12 @@ router.post('/refresh', token.refreshEndpoint);
 
 router.use('/dashboard', dashboard);
 
+// Catch and send the error message down (instead of just 500: internal server error)
+// Need all 4 params, because that's how express knows this is the error handler
+// eslint-disable-next-line no-unused-vars
+router.use((error, req, res, next) => {
```

Comment:
Is it possible to add a test to verify this? I realize we're basically testing express at that point, but perhaps an integration test for the API in a new/existing failure case?

Reply:
@mennovandiermen I've been thinking about this and I don't think we're setup to test this right now because going to the API endpoints involves being redirected through EDL. We also don't have the tests setup to use the local express server yet. 

I could make another endpoint just for the tests, but that seems like a crappy solution, rather than a real case, or I guess try to get the fake server set up for tests.

Reply:
OK, that all seems out of scope for this ticket, so no worries.